src/components/Navbar/index.tsx

```diff
@@ -1,17 +1,30 @@
 import React from "react";
+import { useQuery } from "@apollo/react-hooks";
 import styled from "@emotion/styled/macro";
 import { useAuthStateContext } from "context/auth";
 import { Layout } from "antd";
 import { EvergreenLogo } from "components/icons";
+import { StyledLink } from "components/styles";
 import { Subtitle } from "@leafygreen-ui/typography";
 import { uiColors } from "@leafygreen-ui/palette";
-import { Link } from "react-router-dom";
+import get from "lodash/get";
+import { Link, useLocation } from "react-router-dom";
 import { routes } from "constants/routes";
+import { useLegacyUIURL } from "hooks/index";
+import { GET_USER } from "gql/queries";
+import { GetUserQuery } from "gql/generated/types";
 
 const { Header } = Layout;
+const { white, gray } = uiColors;
 
 export const Navbar: React.FC = () => {
   const { isAuthenticated } = useAuthStateContext();
+  const location = useLocation();
+  const { pathname } = location;
+  const { data } = useQuery<GetUserQuery>(GET_USER);
```

Comment:
Is this making a query to the local cache for user data? if not this would be the third network request made for user data.

Reply:
this query will check the cache before making a network request. `options.fetchPolicy` defaults to `cache-first`.

Reply:
Yeah it defaults to checking the cache first.

Reply:
I made this ticket which will reduce GetUser dispatches to 1. https://jira.mongodb.org/browse/EVG-8165